tests/roles/pull_openstack_configuration/defaults/main.yaml

```diff
@@ -0,0 +1,3 @@
+dpa_dir: "/home/zuul/src/github.com/openstack-k8s-operators/data-plane-adoption"
+os_diff_data_dir: "/home/zuul-worker/src/github.com/openstack-k8s-operators/os-diff"
```

Comment:
Sorry for my late response.
I just updated the PR but since the package are still not ready we are in a mix-spot where the binary and the config file is coming from the github-repo.
With rpm we will have the binary in /usr/bin available and the config file in /etc/os-diff and the github source path could be dropped.

Reply:
I have put relative path but I'm not sure, in the Zuul environment the relative paths will work.